test_mask_upload.py

```
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+"""
+Test script for mask upload functionality
+This script tests the img2img server with mask upload to verify the implementation works.
+"""
+
+import requests
+import json
+import os
+from PIL import Image
+import io
```

Ruff (0.11.9)

`io` imported but unused

Remove unused import: `io`

(F401)